packages/babel-generator/test/fixtures/comments/import-reflection-inner-comment/output.js

```diff
@@ -1 +1 @@
-/* 0 */import /* 1 */module /* 2 */from /* 3 */ from /* 4 */"./module.wasm" /* 5 */;
\ No newline at end of file
+/* 0 */import /* 1 */ module /* 2 */from /* 3 */ from /* 4 */"./module.wasm" /* 5 */;
```

Comment:
There is an inconsistency in the spaces here, but there have been similar ones in the past, and ignoring them should be fine.

Reply:
Yeah I tried to fix some of them, but they have different causes and I couldn't figure out how ti fix _all_ of them :sweat_smile: